group-actors/src/main/java/org/sunbird/service/impl/MemberServiceImpl.java

```diff
@@ -32,13 +36,62 @@ public static MemberService getInstance() {
 
   @Override
   public Response addMembers(List<Member> member) throws BaseException {
+    member.forEach(m -> m.setStatus(JsonKey.ACTIVE));
```

Comment:
please check to set all the operations in one foreach loop

Reply:
Done